cmd/maya-apiserver/app/server/restore_endpoint_v1alpha1.go

```diff
@@ -62,7 +64,7 @@ func (rOps *restoreAPIOps) create() (interface{}, error) {
 	}
 
 	// namespace is expected
-	if len(strings.TrimSpace(restore.Namespace)) == 0 {
+	if !restore.Spec.Local && len(strings.TrimSpace(restore.Namespace)) == 0 {
```

Comment:
should we throw error if Local and NameSpace is NOT 0?

Reply:
In case of local We are not using namespace. We are ignoring it so skipped the check.